socorro/external/boto/crash_data.py

```diff
@@ -72,7 +72,8 @@ def get(self, **kwargs):
                 return get(params.uuid, name=params.name)
             else:
                 return get(params.uuid)
-        except CrashIDNotFound:
+        except CrashIDNotFound as cidnf:
+            self.config.logger.error('%s not found: %s' % (params.datatype, cidnf))
```

Comment:
Since this block throws away the exception and all interesting information about it, this logs at least the message so we can see issues in the logs.